### PR TITLE
Add 2020 linux kernel security review

### PR DESCRIPTION
**Pull request by Amir-Montazery**

### CODE REVIEW

File: reviews/linux-kernel.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+## Front Metadata
```
Comment (scovetta) [inline reviews/linux-kernel.md:1]: Sorry, the template isn't clear on this, but the review needs to start with this text. I added both OSTIF and Atredis as reviewers, and made an assumption that neither are directly associated with the kernel project (full disclosure / bias), and that any compensation received from doing the review wasn't coming directly from the kernel project.

```
---
Publication-State: Active
Access: Public
Reviewers:
- Organization: Open Source Technology Improvement Fund
  Associated-With-Project: false
  Compensation-Source: External
- Organization: Atredis Partners
  Associated-With-Project: false
  Compensation-Source: External  
Domain: Security
Methodology:
- External-Review
Issues-Identified: Non-Severe
Package-URLs:
- pkg:generic/linux-kernel?download_url=https://kernel.org
- pkg:github/torvalds/linux
Review-Date: 2021-01-15
Scope: Non-Implementation
Schema-Version: 1.0
SPDX-License-Identifier: CC-BY-4.0
---
```

Comment (Amir-Montazery) [reply reviews/linux-kernel.md:1]: Thank you @scovetta for making those updates. Your assumptions are indeed correct. The review was sponsored by Linux Foundation. Sorry if the format is wonky as well, I thought to add a folder "linux kernel" similar to npm and zlib but wasn't sure how. Please let me know if there's anything I can do. Thanks again!

**Review by scovetta — APPROVED** [review]
LGTM

**Comment by scovetta** [comment]
@Amir-Montazery Let me know if this is good to go from your end and I'll merge.

**Comment by Amir-Montazery** [comment]
Looks good to go! Thank you!

**Pull request merged**